scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/component/parameter/defaults/SimpleLanguageDeterminer.scala

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nussknacker.engine.definition.component.parameter.defaults
+
+import pl.touk.nussknacker.engine.api.definition._
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+trait SimpleLanguageDeterminer {
+
+  def calculateDefaultValue(
+      determinedEditor: Option[ParameterEditor],
+      defaultValue: Option[String]
+  ): Option[Expression] =
+    for {
+      value <- defaultValue
+      language <- determinedEditor
+        .collect {
```

Comment:
IMO it's better to avoid using `collect` in favor of `map` - in the latter case, in the future, when a new editor is added, a compiler shows us this place, and a developer should consider what to do in this new case (currently, we assume that all new cases will have language SpEL)

Reply:
fixed